jcli/src/jcli_lib/transaction/set_valid_until.rs

```diff
@@ -0,0 +1,22 @@
+use crate::jcli_lib::transaction::{common, Error};
+use jormungandr_lib::interfaces;
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+#[structopt(rename_all = "kebab-case")]
+pub struct SetValidUntil {
+    #[structopt(flatten)]
+    pub common: common::CommonTransaction,
+
+    /// the slot this transaction should be valid until, for example 3.14
+    #[structopt(name = "VALID_UNTIL")]
```

Comment:
What's the renaming for, the kebab-case should work already?
This maybe needs `value_name = "BLOCKDATE"`